lib/cbus_elixir/app/meetings.ex

```diff
@@ -0,0 +1,31 @@
+defmodule CbusElixir.App.Meetings do
+  @moduledoc """
+  Domain operations for meetings.
+  """
+
+  import Ecto.Query, warn: false
```

Comment:
why `warn: false` ?

Reply:
Borrowed the line from `CbusElixir.App` without much thought. Reading the docs, it sounds like that was only there to keep an empty context from warning that no imported functions were used - I'll remove it.